gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/management-openapi-v2.yaml

```diff
@@ -145,6 +145,31 @@ paths:
                     $ref: "#/components/responses/ApisResponse"
                 default:
                     $ref: "#/components/responses/Error"
+    /environments/{envId}/apis/_verify-path:
+        parameters:
+            - $ref: "#/components/parameters/envIdParam"
+        post:
+            parameters:
+                - $ref: "#/components/parameters/envIdParam"
+            tags:
+                - APIs
+            summary: Sanitize API paths
```

Comment:
WDYT to change to something like : Checks and validates the API paths. 

In fact I'm a bit confused.
-`Verify` I'm expecting a return of 200 but no content. 
- `Sanitize` I'm expecting a return to be taken into account before I do my PUT/PATCH. 

Here we have a mix of the 2 🙈 

If we stay with a verify, I don't think it's necessary to return content. Like before. 

I can also see that it seems to be dedicated to checking V4 APIs, but it would be nice if it could also check V2 APIs VirtualHost.  
Maybe super simple like if more than 1 VirtualHost return 200 if only one without host call existing verify ? 
and also check duplicate. 
this would be very practical and would help with the mapiV2 migration. 🙊

Reply:
Yes this is intended to manage V2 APIs too. I think this is confusing that the parameter is called PathV4 but indeed it can be used also for V2 APIs. Maybe I should rename `PathV4` to `Path` or even create a new object. WDYT ?

Reply:
⚠️ PathV4 are dedicated to V4 APIs. If we want to handle both kind of APIs, we must have some sort of `oneOf` VirtualHost(V2) & PathV4 (v4)

Reply:
Why not creating a new object `Path`? This is a type that will be used only by a request payload so I don't see the point of using `oneOf` here

Reply:
Yes you're right, I thought we already have a `Path` object.

Reply:
I created a new object PathToVerify (we don't need to `overrideAccess` for the verification so it is a simplified Path)